masterlist.yaml

```
@@ -10196,6 +10359,16 @@ plugins:
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/12711/' ]
     msg: [ *patch3rdParty_SRPC ]
 
+  - name: 'FS_TeleportTomes_Cities.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/43666/' ]
+    msg: [ *patch3rdParty_SRPC ]
+  - name: 'FS_TeleportTomes_FactionsHomes.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/43666/' ]
+    msg: [ *patch3rdParty_SRPC ]
+  - name: 'FS_TeleportTomes_TownsVillages.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/43666/' ]
+    msg: [ *patch3rdParty_SRPC ]
```

These can be combined with a regex.
```suggestion
  - name: 'FS_TeleportTomes_(Cities|FactionsHomes|TownsVillages)\.esp'
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/43666/' ]
    msg: [ *patch3rdParty_SRPC ]
```